module-apps/application-phonebook/widgets/InputLinesWithLabelWidget.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include "application-phonebook/data/PhonebookInternals.hpp"
+#include "application-phonebook/data/PhonebookItemData.hpp"
+#include "application-phonebook/widgets/ContactListItem.hpp"
+
+#include <ListItem.hpp>
+#include <Text.hpp>
+#include <TextFixedSize.hpp>
+
+namespace gui
+{
+    class InputLinesWithLabelWidget : public ContactListItem
+    {
+      public:
+        explicit InputLinesWithLabelWidget(
+            phonebookInternals::ListItemName listItemName,
+            const std::function<void(const UTF8 &text, bool emptyOthers)> &navBarTemporaryMode = nullptr,
```

Comment:
bool params are not too meaningfull. consider enum instead

Reply:
well that is propagated across whole system for navbar :) changing it will touch a lot of files so not in that PR.